guides/common/modules/proc_assigning-multiple-content-view-environments-to-activation-keys.adoc

```diff
@@ -0,0 +1,15 @@
+[id="Assigning_multiple_content_view_environments_to_activation_keys_{context}"]
+= Assigning multiple content view environments to activation keys
+
+You can assign multiple content view environments to an activation key to provide hosts with access to content from each assigned environment.
+Hosts registered with the activation key receive the environments in the order they are stored.
+For more information, see xref:content-view-environment-ordering-and-priority[Content view environment ordering and priority]. 
+
+.CLI procedure
```

Comment:
Should we add a prerequisite here? The allow_multiple_content_views setting must be enabled.